test/cell_test.rb

```diff
@@ -0,0 +1,113 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/ship'
+require './lib/cell'
+
+class CellTest < Minitest::Test
+
+  def setup
+    skip
+    @cell = Cell.new("B4")
+  end
+
+  def test_it_exists
+    skip
+    assert_instance_of Cell, @cell
+  end
+
+  def test_it_has_coordinates
+    skip
+    assert_equal "B4", @cell.coordinate
+  end
+
+  def test_it_does_not_have_a_ship
+    skip
+    assert_equal nil, @cell.ship
```

Comment:
@amymoller 
You don't have to change this at all - this is just a tip. 
Instead of `assert_equal nil, @cell.ship` you could also use `assert_nil` which does the same thing!
So: `assert_nil @cell.ship`

Reply:
Cool! Good to know.